addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -330,6 +336,22 @@ var Wysiwyg = Widget.extend({
         if (reload) {
             window.location.reload();
         }
+    }
+    ,discardEditions: async function (reload = true) {
+        var self = this;
+        return new Promise(function (resolve, reject) {
+            var confirm = Dialog.confirm(this, _t("If you discard the current edits, all unsaved changes will be lost. You can cancel to return to edit mode."), {
+                confirm_callback: resolve,
+            });
+            confirm.on('closed', self, reject);
+        }).then(function () {
+            self.trigger_up('edition_will_stopped');
+            self.trigger_up('edition_was_stopped');
```

Comment:
Why do you trigger theses?
I don't think they are necessary when discarding changes